docs/static/dead-letter-queues.asciidoc

```diff
@@ -1,14 +1,20 @@
 [[dead-letter-queues]]
-=== Dead Letter Queues
-
-NOTE: The dead letter queue feature is currently supported for the
-<<plugins-outputs-elasticsearch>> output only. Additionally, The dead
-letter queue is only used where the response code is either 400
-or 404, both of which indicate an event that cannot be retried.   
-Support for additional outputs will be available in future releases of the 
-Logstash plugins. Before configuring Logstash to use this feature, refer to
-the output plugin documentation to verify that the plugin supports the dead
-letter queue feature.
+=== Dead Letter Queues (DLQ)
+
+[NOTE] 
+--
+The dead letter queue is currently supported only for the
```

Comment:
There are two levels of abstraction here that may be benefited from teasing apart:

 - First, the HTTP request _itself_ can fail (whether that is because Elasticsearch is unreachable or it returns an error HTTP status code), and in these cases the Elasticsearch Output will retry the _entire_ request indefinitely; DLQ has no opportunity to intercept.
 - Second, the HTTP request can succeed (`HTTP/1.1 200 OK`), but the response body can include metadata indicating that one or more specific actions inside the bulk request could not be performed, along with an HTTP-style status code per entry to indicate _why_ the action could not be performed (so we have similar 400 indicating malformed input, 404 not found, etc.), and it is in _this_ case that individual indexing failures can be routed to the DLQ.